index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anwar and mani</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 id="title">Survey form</h1>
+    <p id = "description">This is the solution for survey form page on Microverse trial. Which is solved by Anwar and mani.</p>
```

Comment:
Please create a header tag and put it inside this h1 and p tag, so look like professional and clean code 👍